src/sass/pages/_signUp.scss

```diff
@@ -0,0 +1,94 @@
+.signUp {
+  display: flex;
+
+  @include respond(phone) {
+    flex-direction: column;
+  }
+
+  &__left,
+  &__right {
+    width: 50%;
+    @include respond(phone) {
+      width: 100%;
+    }
+  }
+
+  &__left {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  &__right {
+    background: $light-blue;
+    position: absolute;
+    top: 0;
+    right: 0;
```

Comment:
this should not be necessary if you follow a just flex approach without using position for this. For the svg's positioning its ok for the play button that shows up in the front of the phone svg

Reply:
Actually it is because we need the phone app section to be over the header. 
This is the best I came up with.